Makefile

```diff
@@ -175,6 +178,15 @@ else
 	@echo "kustomize has been installed"
 endif
 
+install-prometheus:
+	@echo "installing prometheus"
+	# install prometheus
+	helm install ${PROMETHEUS_NAME} stable/prometheus-operator
+	# install service monitor
+	@echo "prometheus has been installed"
```

Comment:
please add installing service monitor here => something similar to `kustomize build config/prometheus | kubectl apply -f - `

Reply:
As per the earlier comment, if we do that then we lose the ability to automatically inherit the namespace, name prefixing, service resolution etc which would require duplicating config in kustomize.

Reply:
I understand the double up and I'm happy with it